src/main/java/dev/aura/bungeechat/config/Configuration.java

```diff
@@ -332,15 +332,20 @@ private void convertYAMLConfig() {
             section = modulesSection.getSection("WelcomeMessage");
             final ImmutableMap<String, Object> moduleWelcomeMessage = ImmutableMap.<String, Object>builder()
                     .put("enabled", section.getBoolean("enabled")).build();
-
+            
+            section = modulesSection.getSection("LocalTo");
```

Comment:
This is for converting the old config only. The new section should have been just created in the config. This class doesn't need any changes.